pytomorrowio/const.py

```diff
@@ -21,6 +21,8 @@
 TYPE_SOLAR = "solar"
 TYPE_PRECIPITATION = "precipitation"
 
+MAX_FIELDS = 55
```

Comment:
It's not possible to request more than `MAX_FIELDS` for any given `timestep` value. I'm adding a test case for that.